components/camel-pdf/src/main/java/org/apache/camel/component/pdf/PdfConfiguration.java

```diff
@@ -68,9 +67,9 @@
     @UriParam(defaultValue = "14")
     private float fontSize = 14;
     @UriParam(defaultValue = "A4", enums = "LETTER,LEGAL,A0,A1,A2,A3,A4,A5,A6")
-    private PDRectangle pageSize = PDRectangle.A4;
+    private String pageSize = PAGE_SIZE_A4;
     @UriParam(defaultValue = "Helvetica")
-    private PDFont font = PDType1Font.HELVETICA;
+    private String font = "Helvetica";
```

Comment:
Can you add the font names as enums = "xxx,yyy" so we have them there for tooling and whatnot